README.md

```diff
@@ -89,203 +59,101 @@ You can specify the project files in `polymer.json` so that commands like `polym
 
-## Commands
 
-### help
+### `polymer init [TEMPLATE]`
 
-Displays help on commands and options:
+Initializes a Polymer project from one of several templates. Pre-bundled templates range from just bare-bones to fully featured applications like the [Polymer Starter Kit](https://github.com/PolymerElements/polymer-starter-kit).
 
-    $ polymer help
+You can download and run templates built by our community as well. [Search npm](https://www.npmjs.com/search?q=generator-polymer-init) for a template you'd like to use. Then install it and the CLI will pick it up automatically.
 
-### init
+Run `polymer init` to choose a template from a list of all installed templates. Or if you know the template name before hand, you can provide it as a command argument.
 
-Initializes a Polymer project from one of several templates.
 
-Choose a template from a menu:
+### `polymer install [--variants]`
 
-    $ polymer init
+Install your dependencies, similar to running `bower install`.
 
-Create a new project from the 'element' template:
+If the `--variants` option is provided, the command will also search your project's `bower.json` for  a `"variants"` property and install any [dependency variants](https://www.polymer-project.org/2.0/docs/glossary#dependency-variants) listed there as well as well. This is useful if you need to test your elements against multiple versions of Polymer and/or sets of dependencies.
 
-    $ polymer init element
 
-You can download and run templates built by our community as well. [Search npm](https://www.npmjs.com/search?q=generator-polymer-init) for the full list.
+### `polymer serve [options...]`
 
-### install
+Start a development server designed for serving Polymer & Web Component projects. Applications are served as-is, while elements are served from a special route where it can properly reference its dependencies.
 
-Installs Bower dependencies, optionally installing multiple variants.
+Run `polymer help serve` for the full list of available options.
 
-    $ polymer install
 
-This performs a Bower install of dependencies listed in `bower.json`, and is
-equivalent to `bower install`.
+### `polymer lint [--rules RULE_SET] [options...]`
 
-    $ polymer install --variants
+Lint your project for common errors. Provide a set of linting rules via either your `polymer.json` configuration file or the `--rules` command option.
 
-This performs a Bower install, and also installs any dependency variants
-specified in the `"variants"` property of `bower.json`. See
-[Dependency Variants](#dependency-variants) above.
-
-### lint
-
-Add a "lint" section to your `polymer.json` file at the root of your project:
+For example, if you wanted to lint a Polymer 2.0 Hybrid element, you would run `polymer lint --rules polymer-2-hybrid`. To make sure the correct rule set is always used, you should add the following field to your `polymer.json`:
```

Comment:
To make sure the correct rule set is always used, you can add a "lint" section to your `polymer.json` like so: